src/sentry/workflow_engine/handlers/condition/event_frequency_handlers.py

```diff
@@ -17,7 +17,8 @@
 @condition_handler_registry.register(Condition.EVENT_FREQUENCY_COUNT)
 @condition_handler_registry.register(Condition.EVENT_UNIQUE_USER_FREQUENCY_COUNT)
 class EventFrequencyCountHandler(DataConditionHandler[list[int]]):
-    type = [DataConditionHandler.Type.WORKFLOW_TRIGGER, DataConditionHandler.Type.ACTION_FILTER]
+    type = DataConditionHandler.Type.ACTION_FILTER
```

Comment:
is `type` only used to render FE? currently all existing event frequency conditions are workflow triggers, we will have a migration after rolling out workflow engine to move where these live

Reply:
Yes I think the `type` is only being used for the UI, @saponifi3d could you confirm?

Reply:
👍 yeah, it's just used so we can filter the registry in the API -- that way we can drive different drop downs with different data conditions